utils/parse.ts

```diff
@@ -9,3 +9,9 @@ export const extractTwitterIdFromDescription = (desc: string = '') => {
 
   return ''
 }
+
+export const removeSuffixFromString = (content: string = '') => {
+  const words = content.split(' ')
```

Comment:
Imo it possible to chain fonction (string.split().pop()....)

Reply:
<img width="670" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/e07b1db5-11ec-45c9-82b6-e46e9715fcc1">
It could not be that because the pop function only return the last element.